apps/frontend/src/app/global.css

```
@@ -0,0 +1,413 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+html {
+  -webkit-text-size-adjust: 100%;
+  font-family: ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont,
+    Segoe UI, Roboto, Helvetica Neue, Arial, Noto Sans, sans-serif,
+    Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol, Noto Color Emoji;
```

Biome (2.1.2)

[error] Duplicate font names are redundant and unnecessary: Segoe

Remove duplicate font names within the property

(lint/suspicious/noDuplicateFontNames)